serverLib.sml

```diff
@@ -369,21 +369,23 @@ structure GitHub = struct
     end
 end
 
+
+ok%  
+
 structure Slack = struct
-  val token = until_space (file_to_string "slack-token")
-  val channel = "CCKESLBJQ"
-  val postMessage_endpoint = "https://slack.com/api/chat.postMessage"
+  val url = until_space (file_to_string "custom-uri")
+  val postMessage_endpoint = "https://hooks.slack.com/services/"
   fun postMessage_curl_cmd text = (curl_path,["--silent","--show-error",
-    "--header",String.concat["Authorization: Bearer ",token],
-    "--header","Content-type: application/json;charset=utf-8",
     "--request","POST",
+    "--header","Content-type: application/json;charset=utf-8",
     "--data",String.concat["{\"channel\":\"",channel,"\",",
```

Comment:
channel got deleted?

Reply:
Yes. The bot can only send to the „worker-admins“ Channel as per the web configuration. Slack required picking a channel for web apps now.

Reply:
yeah but you're still using the variable that got deleted